### DISCUSSION TITLE
n8n部署不成功

### DISCUSSION THREAD

**Discussion by oicqq** [zeabur/.github#16] [Q&A] [answered]
n8n部署不成功，要如何部署？

**Comment by yuaanlin** [comment]
您好，請到我們的 Discord 官方社群 https://discord.gg/DrdGCvXEyY 取得即時的技術支援服務，謝謝！

**Comment by MichaelYuhe** [comment, marked as answer]
您好这是部署文档：https://zeabur.com/docs/marketplace/n8n ，后续问题请移步 Discord 社群
